Assets/Scripts/Login/Networking/LoginServerClient.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using System.IO;
+using System.Net;
+using Login.Data;
+using UnityEngine;
+using Utilities;
+
+namespace Login
+{
+    /// <summary>
+    /// Class responsible for communicating with the Login API.
+    /// Contains all supported endpoints and any special logic required to serialize/deserialize requests/responses
+    /// </summary>
+    public class LoginServerClient
+    {
+        public static readonly string prefix = "/api/";
```

Comment:
This is very debatable, but I never end urls with an `/`. That way you always know that when you add more you need to add the `/` too